nsqworker/nsqhandler.py

```diff
@@ -128,7 +128,7 @@ def flock(self, message):
 
 class NSQHandler(NSQWriter):
     def __init__(self, topic, channel, timeout=None, concurrency=1, max_in_flight=1,
-                 message_preprocessor=None, service_name=get_random_string()):
+                 message_preprocessor=None, service_name=get_random_string(), raven_client=None):
```

Comment:
is it an over kill to pass an object over here? instead of so many params?

Reply:
It makes it a breaking change and I don't think it worth it

Reply:
coolz